src/gen_mod.erl

```diff
@@ -84,23 +79,24 @@
 -include("mongoose.hrl").
 
 -record(ejabberd_module, {
-          module_host, % {module(), jid:server()},
+          module_host_type, % {module(), host_type()},
           opts % list()
          }).
 
 -type module_feature() :: atom().
+-type domain_name() :: jid:server().
+-type host_type() :: binary().
```

Comment:
There is `mongoose:host_type()` already.

Reply:
yes, right. i started work on this PR before Mikhail introduced `mongoose:host_type()` type, and then just forgot to change it after rebasing.